Config.uk

```diff
@@ -45,6 +45,15 @@ config LWIP_UNIKRAFT21X
 endchoice
 
 menu "Netif drivers"
+config LWIP_LOOPIF
+	bool "Loop interface"
+	default n
+	implies LWIP_LOOPBACK
```

Comment:
I think this should be `depends on` instead of `implies`. Using `implies` seems to generate build errors.

Reply:
Hum :thinking:, I thought I have tested this case, anyways. But it makes anyways sense so I will push an update with `select` that will enable loopback traffic as soon as the loop interface is enabled. I find `depends on` confusing because if you do not enable loopback traffic the loopback interface will not appear in the interfaces list.